editor/include/ImGuiCore/TypeTemplate/Template.hxx

```diff
@@ -21,7 +21,20 @@ void draw(double &value, const std::string &name) { ImGui::InputDouble(name.c_st
 
 void draw(pivot::ecs::data::Asset &asset, const std::string &name) { ImGui::InputText(name.c_str(), &asset.name); }
 
-void draw(pivot::ecs::data::Void &, const std::string &) { ImGui::Text("This component has no data"); }
+void draw(pivot::ecs::data::Void &, const std::string &name)
+{
+    ImGui::Text("%s: This component has no data", name.c_str());
+}
+void draw(pivot::EntityRef &value, const std::string &name)
+{
+    // FIXME: Handle entity reference input. Be careful about handling null
+    // references. Empty input should store EntityRef::empty() in the reference.
+    if (value.is_empty()) {
+        ImGui::Text("%s: Empty entity ref", name.c_str());
+    } else {
+        ImGui::Text("%s: Entity ref %u", name.c_str(), value.ref);
```

Comment:
Is it possible to have the Tag Component displayed instead of the Id ?

Reply:
Updated with a text input with the entity tag.